src/components/shared/snackNotification.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div id="ConnectionStatusSnack-ctn">
+    <v-snackbar v-model="visibility">
+      <span id="snack-message">
+        {{ message }}
+      </span>
+
+      <template #action="{ attrs }">
+        <div
+          v-bind="attrs"
+          id="close-icon"
+          @click="handleClose"
+        >
+          <Button
+            icon="close"
+            color="white"
+          />
+        </div>
+      </template>
+    </v-snackbar>
+  </div>
+</template>
+
+<script>
+import { mapState } from 'vuex'
+import AmbButton from '@/components/shared/Button'
+import { PEER_CONNECTED_NOTIFICATION, PEER_DISCONNECTED_NOTIFICATION, PEER_CONNECTING_NOTIFICATION } from '@/components/utils'
+
+export default {
+  name: 'ConnectionStatusSnack',
+  data: () => ({
+    visibility: false,
+    message: 'Connecting to Ambianic Edge device'
+  }),
+  components: {
+    Button: AmbButton
+  },
+  created () {
+    this.setConnectionStatusNotification()
+  },
+  computed: {
+    ...mapState({
+      peerConnectionStatus: state => state.pnp.peerConnectionStatus
+    })
+  },
+  methods: {
+    handleClose () {
+      this.visibility = false
+    },
+    setConnectionStatusNotification () {
+      const lastConnectionStatus = localStorage.getItem('LAST_CONNECTION_STATUS')
+
+      if (lastConnectionStatus !== this.peerConnectionStatus) {
+        switch (this.peerConnectionStatus) {
+          case 'PEER_CONNECTING':
+            this.message = PEER_CONNECTING_NOTIFICATION
+            this.visibility = true
```

Comment:
@vickywane 
`this.visibility = true` can be set once before or after the switch. No need to repeat the same code multiple times without clear benefit.

Reply:
Setting `visibility` once before the switch will mean the snackbar will be displayed for all notification statuses. Currently, the notification is only being shown for connected, connecting and disconnected statuses. 

Alternatively, an if condition can be used to ensure the supported statuses are matched before this method is executed. Would that be better?

Reply:
I have extracted the assignments within each case to a separate method to make follow the DRY convention.